src/content/pages/index.html

```diff
@@ -17,8 +17,8 @@
         <div class="span-1-7 content-container">
             <div class="">
                 <div class="section-head">
-                    <h1 class="visually-hidden">{{title}}</h1>
-                    {% if site.homepage.animation.enabled %}
+					{% if site.homepage.animation.enabled %}
+                    	<h1 class="visually-hidden">{{ site.homepage.title }}</h1>
```

Comment:
This change moves the visually hidden `h1` inside the `if` check, but that's because in the `else`, there's already an unbroken `h1` that's sufficiently exposed to assistive technologies. Moving this keeps us from unnecessarily rendering extra `h1`s.